2024-04-22 16:56:07,591 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 16:57:07,727 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 16:58:07,822 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 16:59:07,904 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:00:08,015 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:01:08,104 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:02:08,210 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:03:08,255 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:04:08,353 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:05:08,461 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:06:08,552 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:07:08,631 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:08:08,709 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:09:08,807 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:10:08,920 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:11:09,034 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:12:09,062 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:13:09,173 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:14:09,279 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-22 17:15:09,382 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

